hw/ip/aon_timer/README.md

```diff
@@ -17,52 +20,63 @@ See that document for an overview of how it is integrated into the top level sys
-* Watchdog pause during system escalation
+Since this timer functions as a watchdog, it has two additional features not present in the always-on wakeup timer:
+* Watchdog configuration lock by writing 0 to register `wdog_regwen`
+* Watchdog pause in sleep is enabled if field `wdog_ctrl.pause_in_sleep` is 1 and the counter will halt the count whenever `sleep_mode_i` is set after it's been synchronised.
 
-Unlike the wakeup timer, the watchdog timer configuration can be locked by firmware until the next system reset.
+Once the watchdog timer configuration is locked, firmware cannot modify the timer configuration registers  until the next system reset.
 This allows the option of preventing firmware from accidentally or maliciously disabling the watchdog.
 
-The "pause in sleep" option controls whether the watchdog timer continues to count in low-power modes.
+The "pause in sleep" option controls whether the timer continues to increase in low-power modes if enabled.
 This allows configurations where the watchdog timer can remain programmed and locked while the device is put to sleep for relatively long periods, controlled by the wakeup timer.
 Without this feature, the watchdog timer might wake up the core prematurely by triggering a watchdog bark.
 
-The "pause during escalation" feature ensures that watchdog bites and barks do not interfere with system escalation behavior.
-If during escalation software configures the system to hang instead of reset, the watchdog bite cannot supersede that decision.
+
+#### AON Watchdog bark
+
+Assumming `wdog_ctrl.enable=1` when the count reaches the bark threshold, a level wakeup signal (`wkup_req_o` in AON domain) is sent to the power manager.
+Note `wkup_req_o` can get set by either counter once the threshold is surpassed.
+In addition, a level IRQ (`intr_wdog_timer_bark_o` in SYS domain) is also generated to the processor.
+If the system is in a low power state, `wkup_req_o` signal asks the power manager to wake the system so that the IRQ (`intr_wdog_timer_bark_o`) can be serviced.
+If the system is not in a low power mode, the IRQ is immediately serviced.
+The level wakup signal (`wkup_req_o`) stays high until there's a system reset or it's explicitly acknowledged by software by writing a 0 to the [`WKUP_CAUSE`](doc/registers.md#wkup_cause) register.
+To clear the level interrupt (`intr_wdog_timer_bark_o`) write 1 to the field [`INTR_STATE.wdog_timer_bark`](doc/registers.md#intr_state).
+The condition of the count reaching the bark threshold is known as the watchdog bark.
+
+An extra interrupt output is available to connect the watchdog bark output to an NMI pin (`nmi_wdog_timer_bark_o`) if required.
+
+#### AON Watchdog bite
+
+The condition of the count reaching the bite threshold is known as the watchdog bite.
+Assumming `wdog_ctrl.enable=1` when the count reaches the bite threshold , a reset request (`aon_timer_rst_req_o` in AON domain) is sent to the power manager which will trigger a system reset.
```

Comment:
Again, I think this needs tweaking. The text currently sounds like we're talking about whether the watchdog was enabled when the counter reached the threshold. It might be clearer to talk about a signal that comes out if the counter is above the threshold and the watchdog is enabled?